pytket/extensions/qiskit/tket_backend.py

```diff
@@ -66,9 +66,8 @@ class TketBackend(BackendV2):
     :py:class:`qiskit.aqua.QuantumInstance`, providing a custom
     :py:class:`qiskit.transpiler.PassManager` with a
     :py:class:`qiskit.transpiler.passes.Unroller`. For examples, see the `user manual
-    <https://tket.quantinuum.com/user-manual/manual_backend.html#embedding-into-
-    qiskit>`_ or the `Qiskit integration example <ht
-    tps://github.com/CQCL/pytket/blob/main/examples/qiskit_integration. ipynb>`_.
+    <https://docs.quantinuum.com/tket/user-guide/manual/manual_backend.html#embedding-into-
```

Comment:
linting check is failing becuse therse lines are too long. Should we disable the linting for this line @cqc-melf

Reply:
These links were not clickable to begin with it seems.

Reply:
I think it should be possible to have the links on multiple line with adding `\` at the end of the line? Have you tried that?

Reply:
I think just using a new line works. The link will not be clickable in the source file but sphinx produces a valid link in the html pages.